### PR TITLE
move power areas closer to industrial areas, fixes #1679

### LINKED ISSUE #1679: Rendering of power areas is too dominant and too close to landuse=quarry
Maybe rendering it like landuse=industrial or with darker shade of purple would be better solution? It is special kind of landuse=industrial. It is especially bad for solar panel farms covering large areas.

Or maybe even stop special rendering? It anyway should be on landuse=industrial

Examples http://www.openstreetmap.org/#map=15/52.5618/13.5887

selection_006

### PR DESCRIPTION
**Pull request by matkoniecz**
I considered using exactly the same style as landuse=industrial but that change would be too significant.

Intention was to make rendering closer to landuse=industrial but at higher zoom keep it differentiable.

The biggest improvement is in case of solar farms mapped as recommended by wiki.

www.openstreetmap.org/?mlat=51.5746&mlon=13.7358#map=13/51.5746/13.7358
solar farm master-_power 10 18 350px master - power 350px

additional examples:

https://cloud.githubusercontent.com/assets/899988/8877598/71feb586-3226-11e5-9c1e-898564b0312c.png
https://cloud.githubusercontent.com/assets/899988/8877600/722c1c2e-3226-11e5-922e-9c8f3026e6f8.png
https://cloud.githubusercontent.com/assets/899988/8877604/7231ba4e-3226-11e5-97f8-5c93373aaab0.png
https://cloud.githubusercontent.com/assets/899988/8877603/72309c9a-3226-11e5-8f4f-d80964857fdb.png
https://cloud.githubusercontent.com/assets/899988/8877602/72306478-3226-11e5-845a-a9cdc195cb50.png
https://cloud.githubusercontent.com/assets/899988/8877605/7231e0f0-3226-11e5-8d14-d21291bee693.png
https://cloud.githubusercontent.com/assets/899988/8877599/72177954-3226-11e5-8091-b80594139253.png
https://cloud.githubusercontent.com/assets/899988/8877601/722e2afa-3226-11e5-92aa-8e51d1c573ae.png
https://cloud.githubusercontent.com/assets/899988/8877606/72450298-3226-11e5-9bf8-396d3b156272.png
https://cloud.githubusercontent.com/assets/899988/8877607/72454a50-3226-11e5-9a1e-57ab589ee72a.png

### CODE REVIEW

**Comment by kocio-pl** [comment]
Examples of Sheffield and Brno are mistaken I guess (I don't see any power generators there), but Fukushima examples and above German pictures (power stations on industrial area) are the most convincing for me, so :+1:.

**Comment by matkoniecz** [comment]
@kocio-pl It affects power=station/generator/sub_station/substation so maybe it is one of other cases.

**Comment by imagico** [comment]
You don't include power=plant and at the moment and power=generator is shown earlier (z=10) than power=substation (z=13).  In general power=plant and power=substation are larger features and power=generator and power=transformer are smaller features.

It also makes sense to show power=substation with an outline early since those are always closed, access restricted areas while power=plant can be public.

**Comment by matkoniecz** [comment]
> You don't include power=plant and at the moment and power=generator is shown earlier (z=10) than power=substation (z=13). In general power=plant and power=substation are larger features and power=generator and power=transformer are smaller features.

This is not changed here and it may be better done separately. Or maybe just drop rendering in industrial colour below z15, as these objects anyway should be on landuse=industrial area.

> It also makes sense to show power=substation with an outline early since those are always closed, access restricted areas while power=plant can be public.

It has outline since z16 and at least ones that I know would anyway have fences that are already displayed on the map. I am not sure is it beneficial to add stronger outline on top of that.

File: landcover.mss [hunk 1]
```diff
@@ -356,11 +357,17 @@
   [feature = 'power_generator'][zoom >= 10],
   [feature = 'power_sub_station'][zoom >= 13],
   [feature = 'power_substation'][zoom >= 13] {
+    polygon-fill: @industrial;
+    [zoom >= 15] {
       polygon-fill: @power;
-      [zoom >= 12] {
-        line-width: 0.4;
-        line-color: darken(@power, 40%);
```
Comment (pnorman) [inline landcover.mss]: should this be `@power-line`?

Comment (pnorman) [inline landcover.mss]: ah. i see, maybe not. but the relationship between power, power-line, industrial, and industrial-line is not obvious

Comment (matkoniecz) [inline landcover.mss]: I also thought about defining `@power`, `@power-line` and `@high-zoom-power` - as separate values, without depending on `@industrial`.

In the end I used this version as it seemed to be easier to customize.

**Pull request merged**